src/sparsezoo/model/utils.py

```diff
@@ -599,9 +599,8 @@ def _copy_and_overwrite(from_path, to_path, func):
 
 def include_file_download_url(files: List[Dict]):
     for file in files:
-        file["url"] = get_file_download_url(
-            model_id=file["model_id"], file_name=file["display_name"]
-        )
+        file["url"] = get_file_download_url(file["download_url"])
```

Comment:
where is this used now?

Reply:
actually, never mind, see it was replacing the previous key

Reply:
The `get_file_download_url` function is only called from this file, I can get rid of that method and move the logic here if you like. That would look something like:
```python
def include_file_download_url(files: List[Dict]):
    for file in files:
        download_url = file["download_url"]
        del file["download_url"]

        # important, do not remove
        if convert_to_bool(os.getenv("SPARSEZOO_TEST_MODE")):
            delimiter = "&" if "?" in download_url else "?"
            download_url += f"{delimiter}increment_download=False"

        file["url"] = download_url
```

Reply:
no need, looks good thanks